capabilities/streamline-change-approval.md

```diff
@@ -44,20 +60,20 @@ The following capabilities will be valuable for you and your team to explore, as
 
 ### [Continuous Integration](/capabilities/continuous-integration.md) - Downstream
 
-Streamlining Change Approval is required to continuously integrate, as it involves frequently integrating code changes and automatically testing them. By adopting Continuous Integration, teams can detect issues early, supporting faster and more reliable change approvals.
+Streamlining Change Approval is required for Continuous Integration, as it involves frequently integrating code changes and automatically testing them. By streamlining change approvals, teams can detect issues early, supporting faster and more reliable code reviews.
```

Comment:
To *continuously* integrate changes, the approval process must be streamlined. A lengthy approval process increases either the number or size of changes waiting to be merged. This creates a review bottleneck that makes continuous integration impractical.